[2025-03-16T04:00:04.190+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T04:00:04.204+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T03:00:00+00:00 [queued]>
[2025-03-16T04:00:04.211+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T03:00:00+00:00 [queued]>
[2025-03-16T04:00:04.211+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T04:00:04.224+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.check_minio_connection> on 2025-03-16 03:00:00+00:00
[2025-03-16T04:00:04.230+0000] {standard_task_runner.py:72} INFO - Started process 1380 to run task
[2025-03-16T04:00:04.234+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.check_minio_connection', 'scheduled__2025-03-16T03:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpz7xarmhl']
[2025-03-16T04:00:04.236+0000] {standard_task_runner.py:105} INFO - Job 155: Subtask bronze_layer_group.check_minio_connection
[2025-03-16T04:00:04.279+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection scheduled__2025-03-16T03:00:00+00:00 [running]> on host c891f3a5df31
[2025-03-16T04:00:04.341+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T03:00:00+00:00'
[2025-03-16T04:00:04.342+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T04:00:04.344+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T04:00:04.347+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T04:00:04.347+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T04:00:04.348+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T04:00:04.784+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-03-16T04:00:04.840+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T04:00:04.841+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.check_minio_connection, run_id=scheduled__2025-03-16T03:00:00+00:00, execution_date=20250316T030000, start_date=20250316T040004, end_date=20250316T040004
[2025-03-16T04:00:04.890+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T04:00:04.897+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
